### DISCUSSION TITLE
Look of new web ui -- edgy or rounded?  Vote!

### DISCUSSION THREAD

**Discussion by u-an-i** [albar965/littlenavmap#779] [Ideas]
I looked at the old web ui again.
Switching to the new ui, I had a more "comfy" feeling with the old ui, in particular with the map look.

I think this is due to the roundedness of the map image.

Now i had the idea: let me try this in the new web ui. So I did.

Here's the result (center), in comparison with old (left) and new (right):

old | result | new
--- | ----- | ---
<img width="240" alt="old" src="https://user-images.githubusercontent.com/84718885/132950820-b9c1d70e-ca8f-4b71-84e7-fe265b8f3522.png"> | <img width="240" alt="new-withround" src="https://user-images.githubusercontent.com/84718885/132950828-d09284b2-cb79-4cd2-a373-1364af07ff2a.png"> | <img width="240" alt="new" src="https://user-images.githubusercontent.com/84718885/132950832-9217b387-6017-403c-b069-596784f94135.png">

click image to enlarge it

What do you like / think ?

**Comment by albar965** [comment]
Round with the bigger shadow looks better IMO.

**Reply by u-an-i** [reply]
i "understand", do you mean "exactly like the center result" or shadow and roundness even more towards the left old look?

Btw.: my development builds only zoom by pixelating a "hight-distance-zoom" version of the map image. Do you know how I can rectify this? Your production builds look good.

**Reply by albar965** [reply]
Center. But this is mostly up to you. 

No idea why the map is pixelated. `make deploy`? CWD is where the `data/maps` folder is? Offline mode in LNM selected?

**Reply by u-an-i** [reply]
SSL wasn't initialising which i found out after i reset the messages. I copied over the dlls from your production release and now maps are fetched from online! :)

**Reply by u-an-i** [reply]
I leave the look as is for now. (Also, after posting the thread, I suddenly had "no feelings" about the rectangular vs the rounded look anymore and thus couldn't determine a significant preference :| ). Anyone is welcome to share her / his thoughts / longings about / for the web ui (here).

**Reply by albar965** [reply]
> SSL wasn't initialising which i found out after i reset the messages. I copied over the dlls from your production release and now maps are fetched from online! :)

I uploaded installer of the one I use to build here: https://www.magentacloud.de/share/9psdyu-eip#$/

**Reply by u-an-i** [reply]
thank you!
